src/os/bluestore/BitmapFreelistManager.h

```diff
@@ -9,14 +9,15 @@
 #include <string>
 #include <mutex>
 
+#include "common/ceph_mutex.h"
 #include "include/buffer.h"
 #include "kv/KeyValueDB.h"
 
 class BitmapFreelistManager : public FreelistManager {
   std::string meta_prefix, bitmap_prefix;
   KeyValueDB *kvdb;
   std::shared_ptr<KeyValueDB::MergeOperator> merge_op;
-  std::mutex lock;
+  ceph::mutex lock = {ceph::make_mutex("BitmapFreelistManager::lock")};
```

Comment:
i think we can either use brace or equal for the default initializer:
```c++
ceph::mutex lock = ceph::make_mutex("BitmapFreelistManager::lock");
```

or 

```c++
ceph::mutex lock{ceph::make_mutex("BitmapFreelistManager::lock")};
```
but no need to use both of them at the same time.